config/crd/bases/vpcresources.k8s.aws_cninodes.yaml

```diff
@@ -3,8 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.0
-  creationTimestamp: null
```

Comment:
Adding a note for reference, creation timestamp was removed for CRD generation starting v11.0 https://github.com/kubernetes-sigs/controller-tools/pull/800